src/astro/databases/base.py

```diff
@@ -33,6 +33,9 @@ class BaseDatabase(ABC):
     _drop_table_statement: str = "DROP TABLE IF EXISTS {}"
     _create_table_statement: str = "CREATE TABLE IF NOT EXISTS {} AS {}"
 
+    illegal_column_name_chars: List[str] = []
+    illegal_column_name_chars_replacement: List[str] = []
```

Comment:
Would be good to add a comment above these two on what are they used for and when we should override it